### DISCUSSION TITLE
Train a custom model hyperparameter tuning.

### DISCUSSION THREAD

**Discussion by sanjuktasr** [dscripka/openWakeWord#121] [Q&A]
I want to train a wake word on a custom phrase but voice independent. I have tried the training with my positive sample dataset that is manually recorded and collected from various sources. for negative samples I am using validation_set_features. Can you please tell me how to approach the problem because I am getting a very high false reject value?

**Comment by dscripka** [comment]
Can you provide some more information about the model? In particular:

1) What is your wake word or phrase?
2) How many positive examples did you create?
3) What data did you use for negative examples, if different from the defaults?
4) What is the performance report you received at the end of training?
5) Under what conditions are you experiencing a high false reject rates?

**Reply by sanjuktasr** [reply]
1. wake word is hey bata
2. I have pre-recorded ~5000 positive samples.
3. validation_set_features.py -> negative examples
4. reports 80%accuracy, false-reject >300 close to 400

**Reply by dscripka** [reply]
1. That wakeword should work reasonably well, but since it only has 3 syllables and relatively common phoneme combinations it may have lower performance compared to more complex phrases.

2. 5,000 examples is on the low side for a well performing model. I would recommend increasing the total size of the positive examples with synthetic data.

3. The validation_set_features.npy is only ~11 worth of audio, it is not intended to be used for training the model. You will need at minimum 1,000 hours of negative data for training. I would recommend starting with the same negative data that is used in the example training notebooks.

4. How are you measuring this false-reject rate? And are this absolute counts on a separate test set?
